components/meter/meter-mixin.js

```diff
@@ -1,7 +1,6 @@
-import { getLocalizationResource } from '../../tools/localization-load.js';
```

Comment:
Can this file be removed? A quick search of GitHub points at nobody using it currently.

Reply:
Yeah, we should be able to remove it.  Do you think we should keep the generator around?  It's currently serving no purpose, but may in the future.

Reply:
Personally I always lean towards removing it and then bringing it back if we end up needing it one day. Yes, we'll probably forget that it exists, but I feel like that's better than having unused code linger around or even worse having something else reference it and never be able to delete it.